controllers/router.js

```diff
@@ -15,33 +14,38 @@ var house = {
 };
 
 // Map routes to controller functions
-module.exports = function(app) {
+module.exports = function () {
```

Comment:
the problem I see with exporting an object is that it becomes a singleton. Whereas a function can be required/imported and each instance is separate. Which is good for unit testing. You don't want the state of the previous instance to interfere with the next. I'm not sure what the functional programming concept is called.